sys-fs/bees/bees-0.10.ebuild

```diff
@@ -1,112 +0,0 @@
-# Copyright 1999-2023 Gentoo Authors
-# Distributed under the terms of the GNU General Public License v2
-
-EAPI=8
-
-inherit linux-info systemd toolchain-funcs
-
-DESCRIPTION="Best-Effort Extent-Same, a btrfs dedup agent"
-HOMEPAGE="https://github.com/Zygo/bees"
-
-if [[ ${PV} == 9999 ]] ; then
-	inherit git-r3
-	EGIT_REPO_URI="https://github.com/Zygo/bees.git"
-else
-	SRC_URI="https://github.com/Zygo/bees/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="amd64 ~arm64"
```

Comment:
By adding 0.10-r1 as ~arch (which is correct) and then immediately dropping 0.10, you're removing all of bees' stable keywords.

See https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades and the section below too.

Reply:
Ah okay, that's how to read that.

Reply:
I'll keep the drop commit for your convenience: It cleans up the old patch file. Either take it, or insert a stabilization commit before it.